lib/mockRequest.js

```diff
@@ -498,7 +514,18 @@ function createRequest(options) {
         }
         this.emit('end');
     };
-    
+
+    /**
+     * Check if the request was an _XMLHttpRequest_.
+     *
+     * @return {Boolean}
+     * @public
+     */
+    defineGetter(mockRequest, 'xhr', function xhr(){
```

Comment:
@eugef 
@howardabrams 
I just found these lines for express, implementing `.xhr`, though it doesn't seem to work 🤔 
https://github.com/howardabrams/node-mocks-http/blob/f19d9c03f87f7e21cdcb6cb6262d5f99fcf1c991/lib/express/mock-request.js#L159-L162

Reply:
indeed `express` is exported at the buttom:
https://github.com/howardabrams/node-mocks-http/blob/master/lib/http-mock.js#L34